bom/pom.xml

```diff
@@ -795,6 +795,16 @@
                 <artifactId>helidon-microprofile-openapi</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.helidon.microprofile</groupId>
+                <artifactId>helidon-microprofile-reactive-streams</artifactId>
+                <version>${helidon.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.helidon.microprofile</groupId>
+                <artifactId>helidon-microprofile-messaging</artifactId>
+                <version>${helidon.version}</version>
+            </dependency>
         </dependencies>
```

Comment:
It would be maybe better to rebase on top of current master, those two dependencies are already present

Reply:
Just I noticed I can get rid of everything except the new helidon/messaging. I'm going to make it simple in a new PR, with a clean history.